Plugins/Encounter/Encounter.cpp

```diff
@@ -190,6 +190,29 @@ ArgumentStack Encounter::SetPlayerTriggeredOnly(ArgumentStack&& args)
     return Services::Events::Arguments();
 }
 
+ArgumentStack Encounter::GetReset(ArgumentStack&& args)
```

Comment:
These two also need to be added to the `REGISTER()` list at the top of the file

Reply:
Oops. Thanks. Added now.